lib/pbench/test/unit/server/test_shell_cli.py

```diff
@@ -234,16 +120,14 @@ def run(args, cwd: Optional[str] = None) -> subprocess.CompletedProcess:
             "init_indexing",
         ]
         assert called == expected_called
-        assert len(commands) == 3, f"{commands!r}"
-        assert commands[0][0] == "pbench-create-crontab"
-        assert commands[1][0] == "crontab"
-        gunicorn_command = commands[2]
+        assert len(commands) == 1, f"{commands!r}"
+        gunicorn_command = commands[0]
         assert gunicorn_command[-1] == "pbench.cli.server.shell:app()", f"{commands!r}"
         gunicorn_command = gunicorn_command[:-1]
         if user_site:
             assert (
                 gunicorn_command[-2:][0] == "--pythonpath"
-            ), f"commands[2] = {commands[2]!r}"
+            ), f"commands[0] = {commands[0]!r}"
```

Comment:
If this assertion fails, then what it prints out is only tangentially related to the failure of the assertion (and that seems unhelpful).  Yes, in the current code, `gunicorn_command` is `commands[0]`, so the print should encompass all the needed information, but, as the code evolves this might not stay the case.  I think the following would be equivalent but more robust (as long as you're changing this code...):
```python
            assert (
                gunicorn_command[-2:][0] == "--pythonpath"
            ), f"gunicorn_command = {gunicorn_command!r}"
```

Ditto lines 146-148.

Reply:
I only adjusted the existing code to account for removing the crontab commands, so this isn't really a fair criticism, even if you're technically correct. 😁 I had considered using the `gunicorn_command` local throughout, and just didn't feel like spending the time. Ah, well.

Reply:
> this isn't really a fair criticism

I'm not so much criticizing the change that you made as much as expressing dissatisfaction with the result (in the hopes that, because we're changing this line already, we can act to make it better).

> at this point `gunicorn_command` has been stripped

That's a good point.  My concern here is that, because the assertion message won't actually (necessarily) have anything to do with the failed assertion expression, having the added message could serve to obscure the problem rather than to illuminate it.  (PyTest does a pretty good job of reporting what is going on when an assertion fails, so I'm kind of questioning the impulse to intercede here with the second argument to `assert`.)  If this assertion fails, Python will print `commands[0]`...will PyTest print _also_ print `gunicorn_command[-2:][0]` (because it's part of the assertion argument)?  If so, I guess we've got everything covered and I'll just wander away again.